packages/api-gateway/src/drivers/http/adapters/places/handlers.js

```diff
@@ -10,10 +10,30 @@ async function postPlace(req, reply) {
   });
 
   return reply.code(201)
-    .header('Content-Type', 'application/json; chartset:utf-8')
-    .send({ msg: `${place_name} has been saved correctly.` });
+    .send({ message: `${place_name} has been saved correctly.` });
+}
+
+async function deletePlace(req, reply) {
+  const { id } = req.body;
```

Comment:
based on https://github.com/Platzi-Master-C9/booking-services/pull/184/files#r857034509
```suggestion
  const { id } = req.params;
```